dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -39,20 +40,27 @@ using SteadyClock = std::chrono::steady_clock;
 
 struct GACRequestInfo
 {
+    KeyspaceID keyspace_id;
     std::string resource_group_name;
     double acquire_tokens;
     double ru_consumption_delta;
 
     std::string toString() const
     {
         return fmt::format(
-            "rg: {}, acquire_tokens: {}, ru_consumption_delta: {}",
+            "keyspace_id: {}, rg: {}, acquire_tokens: {}, ru_consumption_delta: {}",
```

Comment:
Suggest using format
```suggestion
            "keyspace={} rg={} acquire_tokens={} ru_consumption_delta={}",
```
All other TiFlash codes output keyspace_id with keyword "keyspace=xxx"
![image](https://github.com/user-attachments/assets/cd15c86d-e6f7-428d-ae13-2c9c0914e4f6)

Reply:
this method is never used, I will delete it.

Reply:
and other logs has changed to `keyspace={}`